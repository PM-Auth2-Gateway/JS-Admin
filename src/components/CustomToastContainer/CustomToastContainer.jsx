import React from 'react';
import { ToastContainer } from 'react-toastify';

import 'react-toastify/dist/ReactToastify.min.css';

const CustomToastContainer = () => (
  <ToastContainer
    position='top-center'
    autoClose={2500}
    hideProgressBar
    newestOnTop={false}
    closeOnClick
    rtl={false}
    draggable
    bodyStyle={{ color: 'black' }}
  />
);

export default CustomToastContainer;
