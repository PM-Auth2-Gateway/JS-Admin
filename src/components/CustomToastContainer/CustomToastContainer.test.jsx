import { shallow } from 'enzyme';
import CustomToastContainer from './CustomToastContainer';

describe('CustomToastContainer component', () => {
  it('should be rendered', () => {
    const component = shallow(<CustomToastContainer />);
    expect(component).toMatchSnapshot();
  });
});
