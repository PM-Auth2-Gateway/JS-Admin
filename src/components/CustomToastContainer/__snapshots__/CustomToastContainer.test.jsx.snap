// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomToastContainer component should be rendered 1`] = `
<ToastContainer
  autoClose={2500}
  bodyStyle={
    Object {
      "color": "black",
    }
  }
  closeButton={[Function]}
  closeOnClick={true}
  draggable={true}
  draggableDirection="x"
  draggablePercent={80}
  hideProgressBar={true}
  newestOnTop={false}
  pauseOnFocusLoss={true}
  pauseOnHover={true}
  position="top-center"
  role="alert"
  rtl={false}
  transition={[Function]}
/>
`;
